missioncontrol/v0/pass_task_stack.py

```diff
@@ -4,7 +4,12 @@
 
 
 def get_task_stack(uuid):
-    return  Pass.objects.get(uuid=uuid).task_stack.to_dict()
+    try:
+        return  Pass.objects.get(uuid=uuid).task_stack.to_dict()
```

Comment:
I think you can restructure this so it returns an "ObjectDoesNotExist" error, which already has an error handler.
Something like:
```
TaskStack.get(pass__uuid=uuid).to_dict()
```